### PR TITLE
chore(deps): update deps and apollo server to v4

### PR DESCRIPTION
**Pull request by luluhoc**

### CODE REVIEW

**Comment by luluhoc** [comment]
@marcjulian

File: src/gql-config.service.ts [hunk 1]
```diff
@@ -18,7 +18,7 @@ export class GqlConfigService implements GqlOptionsFactory {
       },
       // subscription
       installSubscriptionHandlers: true,
-      debug: graphqlConfig.debug,
+      includeStacktraceInErrorResponses: graphqlConfig.debug,
       playground: graphqlConfig.playgroundEnabled,
```
Comment (marcjulian) [inline src/gql-config.service.ts:22]: Is the playground still supported? I receive the following error when opening [http://localhost:3000/grapqhl](http://localhost:3000/grapqhl)

```
{"statusCode":404,"message":"Cannot GET /grapqhl","error":"Not Found"}
```

File: README.md [hunk 2]
```diff
@@ -59,12 +59,18 @@ Install [Nestjs CLI](https://docs.nestjs.com/cli/usages) to start and [generate
 ```bash
 npm i -g @nestjs/cli
 ```
+```bash
+yarn add -g @nestjs/cli
```
Comment (marcjulian) [inline README.md:63]:
```suggestion
# npm
npm i -g @nestjs/cli
# yarn
yarn add -g @nestjs/cli
```

File: README.md [hunk 3]
```diff
@@ -59,12 +59,18 @@ Install [Nestjs CLI](https://docs.nestjs.com/cli/usages) to start and [generate
...
+```
 
 Install the dependencies for the Nest application:
 
 ```bash
 npm install
 ```
+```bash
+yarn install
```
Comment (marcjulian) [inline README.md:72]:
```suggestion
# npm
npm install
# yarn
yarn install
```

**Review by marcjulian — CHANGES_REQUESTED** [review]
Thanks looks good, but I will stick to npm and will readd package-lock.json. But we can add info for yarn to the readme.

Can you check if the Graphql Playground still works or if we can enable GraphiQL as the Playground is discontinued.

File: src/gql-config.service.ts (hunk 1 above)
Comment (luluhoc) [reply src/gql-config.service.ts:22]: Yes, it is still supported, it's your url, you have a typo.

**Comment by luluhoc** [comment]
@marcjulian all resolved

File: src/gql-config.service.ts (hunk 1 above)
Comment (marcjulian) [reply src/gql-config.service.ts:22]: True, sorry about that.

**Review by marcjulian — APPROVED** [review]
LGTM!

**Pull request merged**